ACPs/226-dynamic-minimum-block-times/README.md

```diff
@@ -0,0 +1,123 @@
+
+Parameters at activation on the C-Chain are:
+
+<div align="center">
+
+| Parameter | Description | C-Chain Configuration|
+| - | - | - |
+| $M$ | minimum `minimumBlockDelay` value | 100 milliseconds |
+| $q$ | initial target `minimumBlockDelay` excess | 3,141,253 |
+| $D$ | `minimumBlockDelay` update constant | $2^{20}$ |
+| $Q$ | `minimumBlockDelay` update factor change limit | 200 |
+
+</div>
+
+$M$ was chosen to balance providing the option of significantly faster block times without the need for future network upgrades while still providing an upper bound on the maximum possible rate of block production and changes to the `minimumBlockDelay` value.
+
+Based on the 100-millisecond value for $M$, $q$ was chosen such that the effective `minimumBlockDelay` value at time of activation is as close as possible to the current target block rate of the C-Chain, which is 2 seconds.
+
+$D$ and $Q$ were chosen such that it takes approximately 3,600 consecutive blocks of the maximum allowed change in $q$ for the effective `minimumBlockDelay` value to either halve or double.
+
+### ProposerVM `MinBlkDelay`
+
+The ProposerVM currently offers a static, configurable `MinBlkDelay` seconds for consecutive blocks. With this ACP enforcing a dynamic minimum block delay time, any EVM instance adopting this ACP that also leverages the ProposerVM should ensure that the ProposerVM `MinBlkDelay` is set to 0.
+
+### Note on Block Building
+
+While there is no longer a requirement for blocks to burn a minimum block gas cost after the activation of this ACP, block builders should still take priority fees into account when building blocks to allow for transaction prioritization and to maximize the amount of native token (AVAX) burned in the block. 
+
+From a user (transaction issuer) perspective, this means that a non-zero priority fee would only ever need to be set to ensure inclusion during periods of maximum gas utilization.
+
+## Backwards Compatibility
+
+While this proposal requires a network upgrade and updates the EVM block header format, it does so in a way that tries to maintain as much backwards compatibility as possible. Specifically, applications that currently parse and use the existing timestamp field that is denominated in seconds can continue to do so. The `timeMilliseconds` header value only needs to be used in cases where more granular timestamps are required.
+
+## Reference Implementation
+
+A reference implementation is not yet provided, and must be made available for this ACP to be considered `implementable`.
+
+## Security Considerations
+
+Too rapid block production may cause availability issues if validators of the given blockchain are not able to keep up with blocks being proposed to consensus. This new mechanism allows validators to help influence the maximum frequency at which blocks are allowed to be produced, but potential misconfiguration or overly aggressive settings may cause problems for some validators.
```

Comment:
> Too rapid block production may cause availability issues if validators of the given blockchain are not able to keep up with blocks being proposed to consensus.

Can you elaborate on this in the text?

Reply:
By availability do you mean responsiveness for the end-user? Because the term is ambiguous, it can also refer to validator-to-validator availability.